types/ioredis/index.d.ts

```diff
@@ -874,10 +874,13 @@ declare namespace IORedis {
     type ClusterNode = string | number | NodeConfiguration;
 
     interface Cluster extends NodeJS.EventEmitter, Commander {
-        new(nodes: ClusterNode[], options?: ClusterOptions): Redis;
         connect(callback: () => void): Promise<any>;
         disconnect(): void;
-        nodes(role: string): Redis[];
+        nodes: Redis[];
```

Comment:
I don't understand this change, in the code you linked this is a function, not a property?

Reply:
Good catch! I pushed an update to make it a function that accepts an optional `NodeType`.